lib/types/json.js

```diff
@@ -74,7 +76,7 @@ function json (options) {
 
     try {
       debug('parse json')
-      return JSON.parse(body, reviver)
+      return secureJson.parse(body, reviver, { protoAction: protoPoisoning })
```

Comment:
```suggestion
      return secureJson.parse(body, reviver, { protoAction: protoPoisoning, constructorAction: 'ignore' })
```

You have to set `constructorAction` to `'ignore'` to not introduce a breaking change.

Method args: https://github.com/fastify/secure-json-parse?tab=readme-ov-file#sjsonparsetext-reviver-options

Reply:
fixed